plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -333,7 +333,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 			// a role to use for heapster's connections back to the API server
 			ObjectMeta: metav1.ObjectMeta{Name: "system:heapster"},
 			Rules: []rbacv1.PolicyRule{
-				rbacv1helpers.NewRule(Read...).Groups(legacyGroup).Resources("events", "pods", "nodes", "namespaces").RuleOrDie(),
+				rbacv1helpers.NewRule(Read...).Groups(legacyGroup).Resources("events", "pods", "nodes", "nodes/stats", "namespaces").RuleOrDie(),
```

Comment:
@liggitt @deads2k FYI.

Reply:
this looks fine. is metrics needed as well?

Reply:
Kubernetes components (heapster/metrics server) use Metrics API (`/stats/summary`)
`nodes/metrics` gives access to `metrics` subresource that consists of prometheus metrics. 
Authorization code https://github.com/kubernetes/kubernetes/blob/1f4f0123edc973d06ceb15d517eb4c555b944427/pkg/kubelet/server/auth.go#L58